src/elements/content-picker/Footer.scss

```diff
@@ -9,11 +9,23 @@
     justify-content: space-between;
     padding: 0 20px;
 
+    .bcp-footer-left {
+        display: flex;
+        flex: 1;
+        min-width: 0;
+    }
+
     .bcp-selected {
+        flex: 0 0 75px;
         margin: 0;
+        width: 100%;
```

Comment:
Generally not a great idea to mix flex and width. Should the basis or the width take precedence here?